apps/web/src/components/SkillDialog/SkillDialog.stories.tsx

```diff
@@ -42,3 +45,16 @@ LibraryContext.args = {
   context: "library",
   showCategory: false,
 };
+
+export const ShowcaseContext = Template.bind({});
+ShowcaseContext.args = {
+  context: "showcase",
+  showCategory: false,
+};
+
+export const ShowcaseShowMyLibraryContext = Template.bind({});
```

Comment:
I think I am missing something, both new stories look the same

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/44abde1f-6bdb-44dc-b50f-0cf173c46f61)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/8bc7eef6-5673-41d2-99a8-c183f9f40da2)

Reply:
They are pretty darn close but the last one includes a family labelled "My library" which is just those you have added to your skills library (provided by the prop `inLibrary`)

Reply:
Ah, now I get it, it adds an option to the dropdown